### ISSUE TITLE
Change the button style (orange text on white) to have black text

### ISSUE THREAD

**Issue by lache-melvin** [msupply-foundation/open-msupply#5806]
Currently our buttons look like this:
![Screenshot 2024-12-16 at 8 59 32 AM](https://github.com/user-attachments/assets/e5260de3-44d3-4ede-b5e6-d5046768144a)

Orange is not ideal for contrast - necessary text should be in black, just the icon in orange:

![Screenshot 2024-12-16 at 8 59 05 AM](https://github.com/user-attachments/assets/ff14dc9b-027e-48ba-a810-cd112c2d9cdb)

Also update the selectors 🙏 
![Image](https://github.com/user-attachments/assets/1166a4b4-2217-4b4e-b2d5-10c77a360649)

**Comment by lache-melvin** [comment]
Hi @mariyamsupply - can you please let us know what we should do about these variations?

Are the blue text buttons/text ok?
![Screenshot 2024-12-16 at 9 02 46 AM](https://github.com/user-attachments/assets/c831d870-5462-4860-85e4-e1e8676b0e49)

What about the buttons that we have with colour background and white text:

![Screenshot 2024-12-16 at 9 03 08 AM](https://github.com/user-attachments/assets/06c41da5-3fe0-4ba1-8b77-1400e1cde6e5)

![Screenshot 2024-12-16 at 9 02 32 AM](https://github.com/user-attachments/assets/8a5dfbb6-9a65-44d2-8172-e203f9fb49c7)

**Comment by lache-melvin** [comment]
![Image](https://github.com/user-attachments/assets/41c91978-6a35-4e8f-a7c7-41c9c641726c)

@mariyamsupply  also, do you want the selectors (i.e. selected store) to have orange text, or bold like requisition ^

**Comment by lache-melvin** [comment]
blocked waiting on design feedback

**Comment by mariyamsupply** [comment]
Blue text with an arrow indicates a link,
Requisition items don't need orange colour
With colour background buttons I need to describe the behaviour where to use them

**Comment by lache-melvin** [comment]
Decision: ONLY update white buttons with orange text for now -> to black text.

More issues to follow after full button review 🙏

**Comment by aimee-mcneil-melville** [comment]
Question @lache-melvin  @mariyamsupply
When an orange button with white text is hovered on, it changes to a white background with orange text & icon.
Should this hovered version also have the same updated styling (black text, orange icon) or remain as it is?

https://github.com/user-attachments/assets/4a520c26-6f06-4454-85ee-f8588b718143

**Comment by mariyamsupply** [comment]
I'm actually putting together buttons updates, and thought of something a bit nice for hover than bright orange :)
But i'm not sure it sure be a part of this update (as might be standing out from the other hovers)
What do you think?
<img width="887" alt="Screenshot 2024-12-20 at 14 03 05" src="https://github.com/user-attachments/assets/af33dbbb-2d65-46bc-8f5e-cf504795e45b" />

**Comment by aimee-mcneil-melville** [comment]
They look great! I do think if making any changes to colors this should be done as a separate issue. Changing what 'orange' is will change a lot more than just the buttons, and there will be quite a lot more work involved if we need to separate button colors from other colors

I'm just wanting to confirm whether the button that is white background with orange icon/text while it is hovered should also now be black text/orange icon on the white background?

**Comment by lache-melvin** [comment]
Hi @aimee-mcneil-melville - I think leave the orange-background buttons for now, we will create a follow up issue when we have confirmed designs for the other button types 🙏

**Comment by marthakeezy** [comment]
# 🧪 Testing



`V2.5.0 `

These buttons are throughout the app. Some places that have multiple examples are:

Outbound Shipments -> Detail View:
- Top right corner are new style base buttons with icon, hover for orange button with white text & icon
- Select Print: new style flat buttons without icon
- Select More: new style flat buttons with icon (actions)
- Select a shipment: footer has new style flat buttons with icon
- Click delete - see that these buttons have not changed - are blue and white

Settings:
- see the buttons remain orange with white text, and reverse to white button with orange text & icon on hover

Personally, just going to browse through the app and see what's going on with the buttons

**Comment by marthakeezy** [comment]
Cold chain -> Equipment -> Print Asset Label still shows old colours

<img width="330" alt="Image" src="https://github.com/user-attachments/assets/74b9718c-c356-44a9-bd2e-a52a56716493" />

**Comment by marthakeezy** [comment]
Tested in `V2.6.0-test-build-2`

Moving to Done ✅
